src/main/java/de/qabel/desktop/ui/AbstractController.java

```diff
@@ -150,4 +152,29 @@ public Contact gsonContactToContact(GsonContact gc, Identity i) throws URISyntax
 		}
 		return collection;
 	}
+	protected String calculateTimeString(DropMessage dropMessage) {
```

Comment:
Don't think calculateTimeString should care about DropMessages. A `long` parameter should be enough and optimize separation of concerns here.